modules/atomic-widgets/base/atomic-widget-base.php

```diff
@@ -136,33 +132,6 @@ public static function get_props_schema(): array {
 		return static::define_props_schema();
 	}
 
-	// TODO: Move to a `Schema_Validator` class?
-	private static function validate_schema( array $schema ) {
-		$widget_name = static::class;
-
-		foreach ( $schema as $key => $prop ) {
-			if ( ! ( $prop instanceof Atomic_Prop ) ) {
-				Utils::safe_throw( "Prop `$key` must be an instance of `Atomic_Prop` in `{$widget_name}`." );
-			}
-
-			if ( ! $prop->get_type() ) {
-				Utils::safe_throw( "Prop `$key` must have a type in `{$widget_name}`." );
-			}
-
-			$prop_type = AtomicWidgetsModule::instance()->prop_types->get( $prop->get_type() );
-
-			if ( ! $prop_type ) {
-				Utils::safe_throw( "Prop type `{$prop->get_type()}` for prop `$key` does not exist in `{$widget_name}`." );
-			}
-
-			try {
-				$prop->validate( $prop->get_default() );
-			} catch ( \Exception $e ) {
-				Utils::safe_throw( "Default value for `$key` prop is invalid in `{$widget_name}` - {$e->getMessage()}" );
-			}
```

Comment:
@Nevoss 
do you think we should keep this?

Reply:
We should still validate the schema but without validating the default value, maybe?

Reply:
I think validating the default is fine
added the validation back, maybe we'll remove it in the future